pkg/csi/service/vanilla/controller.go

```diff
@@ -232,6 +232,7 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 			multivCenterTopologyDeployment = true
 		}
 		for _, vcenterconfig := range vcenterconfigs {
+			vcenterconfig.ReloadVCConfigForNewClient = true
```

Comment:
It looks like we're only setting the value for ReloadVCConfigForNewClient but never really using it. Do we even need it?

Reply:
removed another private variable `vc.reloadConfigForNewClient`